### PR TITLE
added 2 buttons on landingpage

### PR DESCRIPTION
**Pull request by tracytrle**

### CODE REVIEW

File: src/layouts/DefaultAppBar.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import Stack from "@mui/material/Stack";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import IconButton from "@mui/material/IconButton";
+import MenuIcon from "@mui/icons-material/Menu";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+
+function appBarLabel(label) {
```
Comment (travistrle) [inline src/layouts/DefaultAppBar.js]: Can you update the name to `AppBarLabel` to be consistent with other components? Besides that the file name is DefaultAppBar.js, but non of the function name is matching. Can you update it?

**Review by travistrle — CHANGES_REQUESTED** [review]

File: src/layouts/DefaultAppBar.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <IconButton edge="start" color="inherit" aria-label="menu" sx={{ mr: 2 }}>
+        <MenuIcon />
+      </IconButton>
+      <Typography variant="h6" noWrap component="div" sx={{ flexGrow: 1 }}>
+        {label}
+      </Typography>
+      <Box>
+        <Box sx={{ display: "flex", gap: 2 }}>
+          <Button sx={{ color: "black", backgroundColor: "#bdc3c7" }}>
+            EN
```
Comment (travistrle) [inline src/layouts/DefaultAppBar.js:24]: While we only have 2 languages, I think we could use a for loop here for future extensible.

File: src/pages/Register/RegisterPage.js [hunk 3]
```diff
@@ -4,9 +4,9 @@ import Typography from "@mui/material/Typography";
 import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
 import { useNavigate } from "react-router-dom";
-import { register } from "../api/api";
+import { register } from "../../api/api";
```
Comment (travistrle) [inline src/pages/Register/RegisterPage.js:7]: duplicated?

**Review by travistrle — APPROVED** [review]

File: src/layouts/DefaultAppBar.js (hunk 1 above)
Comment (tracytrle) [reply src/layouts/DefaultAppBar.js]: done

File: src/layouts/DefaultAppBar.js (hunk 2 above)
Comment (tracytrle) [reply src/layouts/DefaultAppBar.js:24]: done

File: src/pages/Register/RegisterPage.js (hunk 3 above)
Comment (tracytrle) [reply src/pages/Register/RegisterPage.js:7]: no

**Pull request merged**